okta/app_filter.go

```diff
@@ -41,13 +41,14 @@ func (a *appFilters) String() string {
 
 func listApps(m interface{}, filters *appFilters) ([]*appID, error) {
 	result := &searchResults{Apps: []*appID{}}
-	qp := &query.Params{Limit: 200, Filter: filters.ApiFilter}
+	qp := &query.Params{Limit: 200, Filter: filters.ApiFilter, Q: filters.getQ()}
```

Comment:
Now that we are using `?q=` for the label, we should document the fact that this will gather results that match via name and label.

Reply:
3e1e5eeb439f6ac1e28c70eaefbe9b8f5aecf75e